axelor-sale/src/main/java/com/axelor/apps/sale/web/SaleOrderLineController.java

```diff
@@ -358,4 +359,18 @@ public void addToCart(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void configuratorDuplicateSaleOrderLine(ActionRequest request, ActionResponse response)
+      throws AxelorException {
+    Context context = request.getContext();
+    SaleOrderLine saleOrderLine =
+        Optional.ofNullable(context.asType(SaleOrderLine.class))
+            // .map(solCtx -> Beans.get(SaleOrderLineRepository.class).find(solCtx.getId()))
```

Comment:
I'm not sure about this being commented, it should save and get the sale order line from the db